app/helpers/template_paths_helper.rb

```diff
@@ -23,4 +23,14 @@ def template_hash_for_member(template, member = nil)
     # hash_for is protected method
     send("hash_for_#{member}#{template_route_prefix(template.class).singularize}_path", :id => template)
   end
+
+  def render_if_exists(partial)
+    partial_exists = lookup_context.find_all(partial).any?
+    if partial_exists
+      render :partial => partial.split("_").last
+    else
+      logger.debug("Did not load partial #{partial} because it could not be found.")
+      nil
```

Comment:
The logger.debug method returns you a boolean indicating if it succeeded or not, the render if exists will place true where the alert is if nil isn't placed.